src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -149,14 +149,22 @@ public function text($message)
     {
         $this->autoPrependText();
 
-        if (!is_array($message)) {
-            $this->writeln(sprintf(' // %s', $message));
-
-            return;
+        $messages = is_array($message) ? array_values($message) : array($message);
+        foreach ($messages as $message) {
+             $this->writeln(sprintf(' %s', $message));
         }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function comment($message)
```

Comment:
I can image a user-case: Having a centralized place where the style is created. And all commands would just work with StyleInterface. Thus one can easily replace the style evewhere. But that is not how commands currently work.